ckanext/restricted/templates/scheming/form_snippets/restricted_fields.html

```diff
@@ -6,7 +6,12 @@
 {% asset 'ckanext-restricted/main_js' %}
 
 {%- set options=[] -%}
-{% set restricted = h.restricted_json_loads(data.get("restricted", field.get("default"))) %}
+
+{%- if data.get('id') -%}
```

Comment:
Do you really need the `-` here in this block? I think they're useful if you creating html only:
https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control